codersdk/deployment.go

```diff
@@ -767,6 +767,9 @@ type ExternalAuthConfig struct {
 	DisplayName string `json:"display_name" yaml:"display_name"`
 	// DisplayIcon is a URL to an icon to display in the UI.
 	DisplayIcon string `json:"display_icon" yaml:"display_icon"`
+	// CodeChallengeMethodsSupported lists the PKCE code challenge methods
+	// The only one supported by Coder is "S256"
```

Comment:
```suggestion
	// The only one supported by Coder is "S256".
```
Nit: Consistency in docs output.